package.json

```diff
@@ -46,10 +48,10 @@
         "cordova-icon": "^1.0.0",
         "cordova-ios": "^4.5.5",
         "cordova-plugin-dialogs": "2.0.1",
-        "cordova-plugin-inappbrowser": "3.0.0",
         "cordova-plugin-network-information": "2.0.1",
         "cordova-plugin-splashscreen": "5.0.2",
         "cordova-plugin-whitelist": "1.3.3",
+        "cordova-remove-cookies": "git+https://github.com/bez4pieci/Phonegap-Cookies-Plugin.git",
```

Comment:
can we reference a specific commit here? getting "whatever is on master" is a bit ugly